docs/design/versioning.md

```diff
@@ -95,27 +95,33 @@ X.Y.Z-beta.W, with an additional +bbbb build suffix added. Furthermore, builds
 that are built off of a dirty build tree, (during development, with things in
 the tree that are not checked it,) it will be appended with -dirty.
 
-### Supported releases
+### Supported releases and component skew
 
 We expect users to stay reasonably up-to-date with the versions of Kubernetes
-they use in production, but understand that it may take time to upgrade.
+they use in production, but understand that it may take time to upgrade,
+especially for production-critical components.
 
 We expect users to be running approximately the latest patch release of a given
 minor release; we often include critical bug fixes in
 [patch releases](#patch-release), and so encourage users to upgrade as soon as
-possible. Furthermore, we expect to "support" three minor releases at a time.
-"Support" means we expect users to be running that version in production, though
-we may not port fixes back before the latest minor version. For example, when
-v1.3 comes out, v1.0 will no longer be supported: basically, that means that the
+possible.
+
+Different components are expected to be compatible across different amounts of
+skew: cluster components (master and node components) should work across two
```

Comment:
"cluster components (master and node components) should work across two minor versions but should be at a version no newer than the master" is confusing because you are grouping the components together but then comparing them to each other. I think it would be clearer to root everything around the master version and then talk about compatibility / skew with that as a reference point.